src/io/DumpH5MD.py

```diff
@@ -0,0 +1,483 @@
+                 store_res_id=False,
+                 static_box=True,
+                 is_adress=False,
+                 author='xxx',
+                 email='xxx',
+                 chunk_size=128,
+                 do_sort=True,
+                 integrator=None):
+        """
+        Args:
+            system: The system object.
+            filename: The name of hdf file name.
+            group_name: The name of atom groups. (default: 'atoms').
+            store_position: If set to True then position will be stored. (default: True)
+            store_species: If set to True then species will be stored. (default: True)
+            store_state: If set to True then state will be stored. (default: False)
+            store_velocity: If set to True then velocity will be stored. (default: False)
+            store_force: If set to True then force will be stored. (default: False)
+            store_charge: If set to True then charge will be stored. (default: False)
+            store_lambda: If set to True then lambda (AdResS) will be stored. (default: False)
+            store_res_id: If set to True then store res_id. (default: False)
+            static_box: If set to True then box is static (like in NVT ensemble) (default: True)
+            is_adress: If set to True then AdResS particles will be save instead of
+                coarse-grained.
+            author: The name of author of the file. (default: xxx)
+            email: The e-mail to author of that file. (default: xxx)
+            chunk_size: The size of data chunk. (default: 128)
+            do_sort: If set to True then HDF5 will be sorted on close.
+            integrator: The integrator, if None then the value from system object is taken.
+        """
+        if not pmi.workerIsActive():
+            return
+        cxxinit(self, io_DumpH5MD, system, is_adress)
+
+        self.filename = filename
+        self.group_name = group_name
+        self.store_position = store_position
+        self.store_species = store_species
+        self.store_state = store_state
+        self.store_velocity = store_velocity
+        self.store_force = store_force
+        self.store_charge = store_charge
+        self.store_lambda = store_lambda
+        self.store_res_id = store_res_id
+        self.static_box = static_box
+        self.chunk_size = chunk_size
+        self.do_sort = do_sort
+
+        self.system = system
+        if integrator is not None:
+            self.integrator = integrator
+        else:
+            self.integrator = system.integrator
+
+        if self.integrator is None:
+            raise RuntimeError('Integrator is not set properly.')
+
+        self.file = pyh5md.H5MD_File(filename, 'w', driver='mpio', comm=MPI.COMM_WORLD,
+                                     creator='espressopp',
+                                     creator_version=espressopp.VersionLocal().info(),
+                                     author=author, email=email
```

Comment:
there's no "email" field in pyh5md.H5MD_File and for this reason no email is printed in the resulting H5MD file. Please change email to author_email, otherwise doesn't get populated.